fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG TMP_BUILD_DIR=/build
```

Comment:
Can you remove this as `ARG`?  You're not intending for it to be an overridden argument.